waiter/src/waiter/auth/authentication.clj

```diff
@@ -29,10 +29,10 @@
 
 ;; An anonymous request does not contain any authentication information.
 ;; This is equivalent to granting everyone access to the resource.
-;; The anonymous authenticator attaches the principal of launch-as-user to the request.
-;; In particular, this enables requests to launch processes as launch-as-user.
+;; The anonymous authenticator attaches the principal of run-as-user to the request.
+;; In particular, this enables requests to launch processes as run-as-user.
 ;; Use of this authentication mechanism is strongly discouraged for production use.
-(defrecord AnonymousAuthenticator [launch-as-user]
+(defrecord AnonymousAuthenticator [run-as-user]
```

Comment:
But it is not authenticating anyone. It is accepting requests from anyone and making them behave as a request from a specific user.